org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -397,9 +397,12 @@ Resources:
                 - prefix: 'arn:aws:s3:::bootstrap-awss3cloudformationbucket-'
                 - prefix: 'arn:aws:s3:::essentials-awss3lambdaartifactsbucket-'
                 - prefix: 'arn:aws:s3:::cf-templates-'
+                # This should only match with 2.1.5.1
+                - 'AWS::::Account:140124849929'
```

Comment:
if this is running on securitycentral you should be able to use `!Sub 'AWS::::Account:${AWS::AccountId}'`